src/components/dashboard/FeedItems/FeedModalItem.js

```diff
@@ -39,6 +42,16 @@ const FeedModalItem = (props: FeedEventProps) => {
   const avatar = get(item, 'data.endpoint.avatar')
   const sellerWebsite = get(item, 'data.sellerWebsite', '')
   const chainId = item.chainId || '122'
+  const direction = useMemo(() => getEventDirection(item), [item])
+  const fromAddress = item?.data?.endpoint?.fromAddress
+  const toAddress = item.data?.endpoint?.toAddress
+
+  let ownerAddress = ''
```

Comment:
owner address is always the same, could be retrieved from wallet.account or wallet.wallet.account (see export wallet data screen as example)

Reply:
Maybe the `ownerAddress` name is a bit misleading.  Its always meant to be the other party of a transaction in the GoodDollar wallet.

The goal is that when any GD wallet user clicks the chat bubble in a given transaction, the user open a direct chat with the other member of that transaction.

We are happy to do in a cleaner way if it exists, but this is not simply just the address of the GD wallet being used.

Reply:
see comment below @cryptoKevinL

Reply:
@izuchukwu-eric  we should just be able to use `data.endpoint.address` instead of determining the counterparty address.  Lets try this out.